### DISCUSSION TITLE
Is the status line only limited to the current buffer?

### DISCUSSION THREAD

**Discussion by nasadorian** [nvim-lualine/lualine.nvim#543] [Q&A] [answered]
Recently converted my vim setup to neovim+lua and am using lualine for status. However there is something I'm confused about - why does the statusline not cover the width of the screen? It seems to follow and fit itself into the focused buffer instead of spanning the screen when splits are present.

Is this just the intended behavior of the statusline? Can I configure it so that the statusline spans the entire screen and highlights the focused buffer?

<img width="1792" alt="Screen Shot 2022-01-10 at 8 28 54 AM" src="https://user-images.githubusercontent.com/1136563/148801720-8b4f23c9-83ad-4505-bcd6-20e0b5615ab7.png">

**Comment by shadmansaleh** [comment, marked as answer]
In vim & neovim statusline is window based . Lualine draws takes separate [config](https://github.com/nvim-lualine/lualine.nvim#default-configuration) for statusline in active & inactive_windows as `sections` & `inactive_sections` respectively .

Currently there's no builtin way to have statusline span entire tabpage width but you can use floatline.nvim ( It draws statusline in a floating window ) . See [here](https://github.com/nvim-lualine/lualine.nvim/issues/504#issuecomment-999532354) for more info.

**Reply by nasadorian** [reply]
Thanks for your response, this is very helpful.

**Comment by nyngwang** [comment]
For future readers: The global statusline feature has been merged on upstream: https://github.com/neovim/neovim/pull/17266.
